### PR TITLE
Fix IOTA SDK links

### LINKED ISSUE #1011: Fix IOTA SDK links
**Describe the bug**

There seems to be a lot of broken stuff related to the IOTA SDK docs

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

I fixed all current broken links, fixed the mess I created by not properly reviewing a PR and fixed the components referencing the SDK.

@thibault-martinez Any `How-Tos` your team would like to promote in the list of links in the component?

We probably also need a better image for the SDK card.
@JSto91 up for a challenge? :trollface: 

## Links to any relevant issues

Fixes #1011 

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/next/components/HomeLayout/CoreLibrariesSection.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React, { FC } from 'react';
 import Link from '@docusaurus/Link';
 import { Shimmer, Wallet } from '@site/src/common/icons';
```
Comment (thibault-martinez) [inline src/next/components/HomeLayout/CoreLibrariesSection.tsx:3]: 'Wallet' is defined but never used

Comment (Dr-Electron) [reply src/next/components/HomeLayout/CoreLibrariesSection.tsx:3]:
```suggestion
import { Shimmer } from '@site/src/common/icons';
```

File: src/next/components/HomeLayout/CoreLibrariesSection.tsx [hunk 2]
```diff
@@ -28,123 +31,56 @@ const Languages: FC<LanguagesProps> = (props) => {
...
           <Languages
             languages={{
-              Rust: '/iota.rs/getting_started/rust',
-              NodeJS: '/iota.rs/getting_started/nodejs',
-              Python: '/iota.rs/getting_started/python',
-              Java: '/iota.rs/getting_started/java',
+              Rust: '/iota-sdk/getting-started/rust',
+              NodeJS: '/iota-sdk/getting-started/nodejs',
+              Python: '/iota-sdk/getting-started/python',
+              Java: '/iota-sdk/getting-started/java',
```
Comment (Dr-Electron) [inline src/next/components/HomeLayout/CoreLibrariesSection.tsx:46]:
```suggestion
```

Comment (Dr-Electron) [inline src/next/components/HomeLayout/CoreLibrariesSection.tsx:46]:
```suggestion
              WASM: '/iota-sdk/getting-started/wasm',
```

**Review by thibault-martinez — APPROVED** [review]

**Review by lucas-tortora — APPROVED** [review]
PreparedCreateNativeTokenTransaction.md seems to be missing. That being said, that's probably a problem for https://github.com/iotaledger/iota-wiki/pull/1009

**Pull request merged**